app/controllers/home_controller.rb

```diff
@@ -1,5 +1,23 @@
 class HomeController < ApplicationController
+  before_action :get_homepage
   def index
     @communities = Community.all
   end
+
+  def carousel_images
```

Comment:
Man liekas tev nevajadzeja atseviško PATCH carousel_images. 
Tev vajadzeja update, kā piemērā https://edgeguides.rubyonrails.org/active_storage_overview.html#has-many-attached

Un tad condition un notices, vai sanaca apdeitot vai ne.

Un tad destroy_carousel_image tev ari nevajag. Tāpēc kā visu laiku tas būs tikai update, un lai izdzēst image, tev vajadzes padot _destroy=true parametru.

Reply:
Sataisiju cik smuki vareju, bet still nevareju nekur iespraust to _destroy, it ka viss strada just fine ari bez vina.